upgrade/1.0.11/scripts/upgrade/ncn-upgrade-cloud-init.sh

```diff
@@ -242,7 +242,6 @@ cat <<EOF>k8s-runcmd-user-data.json
           "/srv/cray/scripts/metal/set-host-records.sh",
           "/srv/cray/scripts/metal/set-dhcp-to-static.sh",
           "/srv/cray/scripts/metal/set-dns-config.sh",
-          "/srv/cray/scripts/metal/set-ntp-config.sh",
```

Comment:
I don't think these need to be yanked.  Loki (1.0.0) has "set-ntp-config" in its boot parameters.   It should just make sure ntp gets synced.